### ISSUE TITLE
Document h5 file structure

### ISSUE THREAD

**Issue by ngroebner** [tsawi/specufex_preprocessing#4]
It would be nice to have an up-to-date reference stating the structure of the h5 files that are created by the different steps. When writing code that uses these files, right now you have to go to the functions that save the files to figure out how they are structured and how to access the data you need. Having a diagram or quick explanation of the hierarchical structure in the github repository (and eventually in a dedicated docmentation site!) would be helpful.

**Comment by tsawi** [comment]
Good idea, @ngroebner :: I think you've mentioned this before, but did you have a recommendation for an existing standardized documentation style?

**Comment by ngroebner** [comment]
There are a few options

- full blown documentation site like for pyasdf
- a docstring in a class that handles the h5 files (which we don't have yet)
- a diagram, table, etc and short explanation in a markdown file on the GitHub repo itself 

The last one is probably a good start just to get something g down on paper. Obviously if you have other ideas we should think about those too.

**Comment by ngroebner** [comment]
I know we have some images of the file structure that were created way at the beginning. Maybe updating these and slapping them on GitHub is enough for now.

**Comment by benholtzman** [comment]
ahh here it is !

**Comment by ngroebner** [comment]
As Long as they're up-to-date, I think that's a great idea.￼
